templates/js-template-default/frameworks/runtime-src/proj.android-studio/app/jni/hellojavascript/main.cpp

```diff
@@ -41,23 +41,29 @@ using namespace cocos2d;
 using namespace anysdk::framework;
 #endif
 
-Application* cocos_android_app_init (JNIEnv* env, int width, int height) {
-    LOGD("cocos_android_app_init");
-    auto app = new AppDelegate(width, height);
+//called when JNI_OnLoad()
+void cocos_jni_env_init (JNIEnv* env) {
+    LOGD("cocos_jni_env_init");
 #if PACKAGE_AS
-    JavaVM* vm;
+    JavaVM *vm;
     env->GetJavaVM(&vm);
     PluginJniHelper::setJavaVM(vm);
 #endif
+}
+
+//called when onSurfaceCreated()
+Application* cocos_android_app_init (JNIEnv* env, int width, int height) {
```

Comment:
add `cocos_jni_env_init` ,and call it at `JNI_OnLoad()`, so that we can init the AnySDK plugin earlier, like creator 1.9.x did.

now `cocos_android_app_init ` was called at `onSurfaceCreated()`. it was called two late, order like:
```
->onCreate(Acticity)
->onStart(Acticity)
->onResume(Acticity)
->setScreenWidthAndHeight(GLSurfaceView.Renderer ,Get Screen Width and Height here)
->onSurfaceCreated(GLSurfaceView.Renderer, native init here, include screen width and height info)
```